src/main/java/de/dennisguse/opentracks/content/data/Track.java

```diff
@@ -177,4 +185,73 @@ public void writeToParcel(Parcel parcel, int i) {
             }
         };
     }
+
+    public static class Selection implements ContentProviderUtils.ContentProviderSelectionInterface {
+        private final List<Id> trackIds = new ArrayList<>();
+        private final List<String> categories = new ArrayList<>();
+        private Instant from;
+        private Instant to;
+
+        private Selection() {
+
+        }
+
+        public static Selection create() {
```

Comment:
What s the benefit of a factory pattern here?

Reply:
I like more the syntax `Selection.create().addTrackId().addCategory()` than `new Selection().addTrackId().addCategory()`.

Anyway, re-reading my argument seems like a stupid one :thinking: :laughing:  so I'm going to change it.